src/util/avro.js

```diff
@@ -1,5 +1,27 @@
+const log = require('./logger').default;
 const avro = require('avsc');
 
+const getPlatformAnalyticsLog = (isGAE = process.env.GAE_INSTANCE) => {
```

Comment:
is there a way to pass an argument instead of using an env var? ideally config is managed by the app.